prometheus/counter.go

```diff
@@ -40,6 +41,16 @@ type Counter interface {
 	// Add adds the given value to the counter. It panics if the value is <
 	// 0.
 	Add(float64)
+	// AddWithExemplar works like Add but also replaces the currently saved
+	// exemplar (possibly none) with a new one, created from the provided
```

Comment:
"(if any)" would be the usual way to word that

Reply:
Yes, much better. Changed.